### ISSUE TITLE
some question about 'MESS' type 

### ISSUE THREAD

**Issue by BourbonTiger** [equinor/ecl-data-io#11]
how can i write a 'MESS' type keyword  like ' STARTSOL' 'ENDSOL' ?

**Comment by eivindjahren** [comment]
Although ecl_data_io correctly reads `MESS` typed fields, it does not support writing it. This should fixed. The `MESS` type doesn't fit neatly into the model of translating the type of the array into a ecl type.

Perhaps it should be signaled by giving some sentinel in the second value?


```
from ecl_data_io import write, MESS


write("file.smry", [("STARTSOL", MESS)])
```

implementing that is just a matter if defining the sentinel and amending `ecl_data_io.types::from_np_dtype`:

```
MESS = UUID
def from_np_dtype(array):
    """
    :param array: A numpy array
    :returns: The corresponding ecl type for the
        given numpy array's dtype.
    """
    if array == MESS:
      return b"MESS"
   ...
```
There will of course need to be some testing and ironing out of details, but this should be the gist of it.

**Comment by eivindjahren** [comment]
Resolved by #14
